docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -0,0 +1,182 @@
+kind: Gateway
+metadata:
+  name: demo-gateway
+  namespace: openshift-ingress
+spec:
+  gatewayClassName: openshift-default
+  listeners:
+  - name: demo
+    hostname: "*.gwapi.${DOMAIN}"
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: All
+EOF
+```
+
+By default, Istio will [automatically provision](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment) a gateway deployment and service with the same name upon creation of this Gateway object:
+```shell
+$ oc get deployment -n openshift-ingress demo-gateway
+NAME           READY   UP-TO-DATE   AVAILABLE   AGE
+demo-gateway   1/1     1            1           25s
+$ oc get service -n openshift-ingress demo-gateway
+NAME           TYPE           CLUSTER-IP       EXTERNAL-IP                          PORT(S)                        AGE
+demo-gateway   LoadBalancer   172.30.175.176   domain.us-east-1.elb.amazonaws.com   15021:30608/TCP,80:31833/TCP   47s
+```
+
+When the Gateway from the previous step is created, the Ingress Operator will automatically create a DNSRecord with the hostname from the listener:
+```shell
+$ oc get dnsrecord -n openshift-ingress
+NAME                         AGE
+gateway-7b49f58c6-wildcard   6m11s
+```
+
+Next, let’s create and configure a namespace called demo-app that will have our HTTPRoute object and our example application:
+```shell
+$ oc create namespace demo-app
+```
+
+Create a demo application deployment and service using `oc new-app`, our cakephp-ingress-demo project, and the foo git branch. This will create a simple server, foo-app, that will allow us to test connectivity. Then use the `oc rollout` command to wait for it to be rolled out:
+```shell
+$ oc new-app -n demo-app --name foo-app https://github.com/openshiftdemos/cakephp-ingress-demo#foo
+$ oc rollout status deployment -w -n demo-app foo-app
+```
+
+Create the HTTPRoute that will direct requests for `demo.gwapi.${DOMAIN}` to our new backend server, foo-app. Note that the parentRefs points to the Gateway, rules specify a `backendRefs` to our service, and hostnames align with the Gateway’s Listeners hostnames:
+```yaml
+$ oc apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http
+  namespace: demo-app
+spec:
+  parentRefs:
+  - name: demo-gateway
+    namespace: openshift-ingress
+  hostnames: ["demo.gwapi.${DOMAIN}"]
+  rules:
+  - backendRefs:
+    - name: foo-app
+      port: 8080
+EOF
+```
+Wait for the gateway deployment to be ready:
+```shell
+$ oc wait -n openshift-ingress --for=condition=ready gateways.gateway.networking.k8s.io demo-gateway
+```
+
+Now let’s curl the route we just created using an HTTP Head request to only get the headers. The app response header should have a value of foo for our demo application. You may have to wait a couple minutes for DNS if your platform uses a domain name for the service’s External IP:
```

Comment:
HTTP verbs are traditionally all upper-case.  I'm not sure whether or not to capitalize "curl" as a verb; one option would be to reword to avoid writing "curl":
```suggestion
Now let’s send a request to the HTTPRoute we just created using an HTTP HEAD request to only get the headers. The `app` response header should have a value of "foo" for our demo application. You may have to wait a couple minutes for the DNS record to propagate if your platform uses a domain name for the service’s External IP:
```
I'm not sure what "if your platform uses a domain name for the service’s External IP" means.  Can that clause be written more clearly?